src/game/input.c

```diff
@@ -2,16 +2,37 @@
 
 #include "specific/s_input.h"
 
+#define DELAY_FRAMES 12
+#define HOLD_FRAMES 3
+
 INPUT_STATE g_Input = { 0 };
 INPUT_STATE g_InputDB = { 0 };
 INPUT_STATE g_OldInputDB = { 0 };
+static int32_t m_HoldBack = 0;
```

Comment:
Let's put a blank above since the vars before are truly global, and these are only modular